packages/node-opcua-server-discovery/source/opcua_discovery_server.ts

```diff
@@ -113,6 +113,16 @@ export class OPCUADiscoveryServer extends OPCUABaseServer {
 
         super(options);
 
+        this._primaryEndpoint = {
```

Comment:
super.options already contains a copy of the options parameter.
No need to duplicate here .